ballerina/tests/47_unary_timestamp_service.bal

```diff
@@ -50,4 +50,26 @@ service "UnaryTimestampService" on ep47 {
             check caller->complete();
         }
     }
+
+    remote function serverStreamTime(TimestampServiceTimestampCaller caller, time:Utc value) returns error? {
+        time:Utc responseTime = checkpanic time:utcFromString("2008-12-03T11:15:30.120Z");
+        time:Utc[] timearr = [responseTime, responseTime, responseTime, responseTime];
+        error? e = timearr.forEach(function(time:Utc val) {
+            checkpanic caller->sendContextTimestamp({
+                headers: {},
+                content: val
+            });
+        });
+    }
+
+    remote function clientStreamTime(TimestampServiceTimestampCaller caller, stream<time:Utc, Error?> clientStream) returns error? {
+        time:Utc[] timearr = [];
+        error? e = clientStream.forEach(function(time:Utc value) {
+            timearr.push(checkpanic time:utcFromString("2008-12-03T11:15:30.120Z"));
```

Comment:
Same, let's replace the `checkpanic`.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/372/commits/69f160eca3668e350a01ca297ba3404aae9ce7d2

Reply:
Fixed the failing issue in https://github.com/ballerina-platform/module-ballerina-grpc/pull/372/commits/3dcc4e846f9601c42174bdf5d63a9766ee306da5